build-tools/packages/build-cli/src/commands/release.ts

```diff
@@ -67,7 +79,8 @@ export class ReleaseCommand<T extends typeof ReleaseCommand.flags> extends State
         this.data.versionScheme = detectVersionScheme(this.data.releaseVersion);
 
         this.data.shouldSkipChecks = flags.skipChecks;
-        this.data.shouldCheckPolicy = flags.policyCheck && !flags.skipChecks;
+        this.data.shouldCheckPolicy =
+            userPolicyCheckChoice ?? (branchPolicyCheckDefault && !flags.skipChecks);
```

Comment:
So `--policyCheck` / `--no-policyCheck` flags trump the `skipChecks` flag?

Reply:
Yes; if the user explicitly passed one of the options it will ignore the skipChecks. I can't think of a case where that's not OK.